### DISCUSSION TITLE
Update comments instead of deleting + creating a new one each time

### DISCUSSION THREAD

**Discussion by utybo** [BetaHuhn/deploy-to-vercel-action#190] [Ideas]
First, thank you so much for creating this Action! Currently using it to deploy a site that gets built in a pretty complex way that wouldn't really work with Vercel's own build system.

The constant deleting + creating of comments can get a bit spammy at times. Would it be possible to add the ability to just update the previous comment the bot made (if any) or create one if this is the first time the PR is deployed?

**Comment by Merott** [comment]
I opened a PR for this: #303
